tests/integration/cluster-users/sar-test.sh

```diff
@@ -240,6 +277,92 @@ function runTests() {
 
 	echo "--> ${VIEW_EMAIL} should NOT be able to create ory Access Rule"
 	testPermissions "create" "rule.oathkeeper.ory.sh" "${NAMESPACE}" "no"
+
+	EMAIL=${NAMESPACE_ADMIN_EMAIL} PASSWORD=${NAMESPACE_ADMIN_PASSWORD} getConfigFile
+	export KUBECONFIG="${PWD}/kubeconfig"
+
+	echo "--> ${NAMESPACE_ADMIN_EMAIL} should be able to create new namespace"
+	createNamespaceForNamespaceAdmin
+	export SHOULD_CLEANUP_NAMESPACE="true"
+
+	# namespace admin should not be able to get or create any resource in system namespaces
+	echo "--> ${NAMESPACE_ADMIN_EMAIL} should NOT be able to list Deployments in system namespace"
+	testPermissions "list" "deployment" "${SYSTEM_NAMESPACE}" "no"
+
+	echo "--> ${NAMESPACE_ADMIN_EMAIL} should NOT be able to get ory Access Rule in system namespace"
+	testPermissions "get" "rule.oathkeeper.ory.sh" "${SYSTEM_NAMESPACE}" "no"
+
+	echo "--> ${VIEW_EMAIL} should NOT be able to create secret in system namespace"
+	testPermissions "create" "secret" "${SYSTEM_NAMESPACE}" "no"
+
+	echo "--> ${NAMESPACE_ADMIN_EMAIL} should NOT be able to delete system namespace"
+	testPermissions "delete" "namespace" "${SYSTEM_NAMESPACE}" "no"
+
+	# namespace admin should be able to get/create k8s and kyma resources in the namespace he created
```

Comment:
```suggestion
	# namespace admin should be able to get/create k8s and kyma resources in the namespace they created
```
Social inequality alert!